From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 19 Dec 2020 09:14:20 +0100
Subject: Decrease Cargo minimal version to 1.46.0

---
 build/moz.configure/rust.configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/moz.configure/rust.configure b/build/moz.configure/rust.configure
index cd77d72..40f017d 100644
--- a/build/moz.configure/rust.configure
+++ b/build/moz.configure/rust.configure
@@ -171,7 +171,7 @@ def rust_compiler(rustc_info, cargo_info):
             )
         )
     rustc_min_version = Version(MINIMUM_RUST_VERSION)
-    cargo_min_version = rustc_min_version
+    cargo_min_version = Version('1.46.0')
 
     version = rustc_info.version
     is_nightly = "nightly" in version.version
